packages/web-components/src/components/pricing-table/pricing-table-header-cell.ts

```diff
@@ -23,6 +23,31 @@ class DDSPricingTableHeaderCell extends StableSelectorMixin(DDSStructuredListHea
   @property({ reflect: true })
   type: PRICING_TABLE_HEADER_CELL_TYPES = PRICING_TABLE_HEADER_CELL_TYPES.COMPLEX;
 
+  @property()
+  private _ctaType?: String;
+
+  private _handleCTASlotChange = ({ target }) => {
+    const cta = target.assignedNodes()[0];
+
+    if (cta) {
+      this._ctaType = cta.getAttribute('cta-type').toLowerCase();
+    }
+  };
+
+  renderCTASlot() {
+    const { _ctaType: ctaType } = this;
+
+    return ctaType === 'video'
```

Comment:
We should probably replace this with the `enum` defined by (I believe) the CTA component. It will look something like this instead:
```
import CTA_TYPE from 'path-to-cta/defs.ts';

...
    
return ctaType === CTA_TYPE.VIDEO
```

I'll work on that.